src/io/CMakeLists.txt

```diff
@@ -32,3 +32,6 @@ target_link_libraries(rs_TFBroadcaster rs_core)
 
 rs_add_library(rs_ResultAdvertiser src/ResultAdvertiser.cpp)
 target_link_libraries(rs_ResultAdvertiser rs_core)
+
+rs_add_library(rs_loadPointCloud src/LoadPointCloud.cpp)
+target_link_libraries(rs_loadPointCloud rs_core)
```

Comment:
ok, you should merge the two and come up with a new PR. no use reinventing the wheel